redwood-treehouse/src/commonMain/kotlin/app/cash/redwood/treehouse/OnBackPressedCallbackService.kt

```diff
@@ -18,11 +18,14 @@ package app.cash.redwood.treehouse
 import app.cash.redwood.ui.OnBackPressedCallback
 import app.cash.zipline.ZiplineService
 import kotlin.native.ObjCName
+import kotlinx.coroutines.flow.Flow
 
 /** Redwood's [OnBackPressedCallback] but implementing [ZiplineService]. */
 @ObjCName("OnBackPressedCallbackService", exact = true)
 public interface OnBackPressedCallbackService : ZiplineService {
   public var isEnabled: Boolean
 
+  public val enabledChangedEvents: Flow<Unit>
```

Comment:
Is there a reason not to go with `StateFlow<Boolean>`?

Reply:
Nope! I was just blinded by getting `OnBackPressedCallbackService` to mimic `OnBackPressedCallback` that I ended up using a `Channel`. Converted to a `StateFlow` and its looking a lot better!